modules/openy_gc_auth/modules/openy_gc_auth_personify/src/Client.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Drupal\openy_gc_auth_personify;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Logger\LoggerChannelFactory;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use GuzzleHttp\Client as HttpClient;
+
+/**
+ * Personify provider client.
+ */
+class Client {
```

Comment:
That is not the exact Personify Client.
Personify has a buggy API and logout request doesn't work with default SOAP API.
That's why Andrey added this custom client.
Let's name it PersonifyLogoutClient for better understanding.

Reply:
Renamed